Secret-Society-Braid/MochiMochiTalk
# Breaking change for heroku integration

- chore: add dep for google drive
- chore: file create
- refactor: change name
- test: add init mocks
- test: add test for new cache algo
- chore: complete the impl
- refactor: adjust git setting
- fix: use StandardCharsets instead
- feat: change dictionary store logic to origin API
- feat: change dict logic to original API
- add bones
- feat: add cacheFileController
- add interface
- feat: add interface for inter class comms
- chore: file added
- feat: add handling action interface
- add methods
- chore: file added
- add impl class
- feat: add thread factory impl
- add impls
- add check method
- feat: complete implementation
- complete implementation for InterClassComms
- fix: fix stream logic
- refactor: organize imports
- chore: file added
- test: add test codes
- test: add test case
- chore: add dir create method
- test: complete test for separating
- fix for extension
- improve usage for CacheFileController#getListOfPaths
- add scheduler for updating list of paths
- chore: file del
- chore: set synchronized for getInstance method
- refactor: organize imports for DeprecatedTTSEngine
- feat: add new logic for voice
- fix for duplicate class
- complete implementation
- add some method
- add some methods
- add line
- complete implementation
- feat: add new tts engine
- chore: add event listener
- update cache and dict stream
